src/widgets/settingspages/HighlightingPage.cpp

```diff
@@ -99,7 +99,7 @@ HighlightingPage::HighlightingPage()
                         .getElement();
 
                 view->addRegexHelpLink();
-                view->getTableView()->horizontalHeader()->hideSection(4);
+                view->getTableView()->horizontalHeader()->hideSection(5);
 
```

Comment:
Your change gets rid of the "Case-sensitive" column but the "Enable regex" column is then shown instead (see image).

![image](https://user-images.githubusercontent.com/35232120/105641497-8a110f80-5e84-11eb-8172-a1995b7a1056.png)

```suggestion
                view->getTableView()->horizontalHeader()->hideSection(
                    HighlightModel::Column::UseRegex);
                view->getTableView()->horizontalHeader()->hideSection(
                    HighlightModel::Column::CaseSensitive);
```
This suggestion hides both columns and also uses the existing `Column` enum which is more readable. :smile: